src/fidesops/service/connectors/saas_connector.py

```diff
@@ -93,8 +93,18 @@ def send(
         try:
             prepared_request = self.get_authenticated_request(request_params)
             response = self.session.send(prepared_request)
-        except Exception:
-            raise ConnectionException(f"Operational Error connecting to '{self.key}'.")
+        except Exception as exc:  # pylint: disable=W0703
+            if config.dev_mode:  # pylint: disable=R1720
+                raise exc
```

Comment:
We might still want to log here which key Fidesops is failing to connect to

Reply:
good improvement, that probably won't be in the trace without adding it.